### ISSUE TITLE
How do I convert a Qimage to s pix

### ISSUE THREAD

**Issue by jkoking** [DanBloomberg/leptonica#588]
Sorry if there is a obvious solution I can't figure it out

**Comment by DanBloomberg** [comment]
You need to find the definition of the QImage class.
The pix is defined in pix.h.
Conversion is typically non-trivial, because you need to take into consideration:
     whether the representation is packed
     if there is padding on each rasterline  (e.g., pix are padded to 32-bit words)
     byte order of pixels (eg., little-endian or big-endian)
     sample order of RGB and A for color images
     colormap conversion if Qimage can use colormaps

**Comment by zdenop** [comment]
What about to try search internet before posting questios?
E.g. https://www.google.com/search?q=qimage+to+leptonica+pix&oq=qimage+to+leptonica+pix&aqs=chrome..69i57j69i60l2j69i61.6828j0j1&sourceid=chrome&ie=UTF-8 -> first found linki is https://stackoverflow.com/questions/8655772/convert-leptonica-pix-object-to-qpixmap-or-other-image-object 
Your problem was solved 10 year ago.

**Comment by jkoking** [comment]
I just wanted to say Zdnop is right fishing not have posted this issue I'm sorry

**Comment by jkoking** [comment]
> What about to try search internet before posting questios?
> E.g. https://www.google.com/search?q=qimage+to+leptonica+pix&oq=qimage+to+leptonica+pix&aqs=chrome..69i57j69i60l2j69i61.6828j0j1&sourceid=chrome&ie=UTF-8 -> first found linki is https://stackoverflow.com/questions/8655772/convert-leptonica-pix-object-to-qpixmap-or-other-image-object 
> Your problem was solved 10 year ago.

You're right I should not have posted this issue I do not know what I was thinking I'm sorry
